### PR TITLE
add presets and new filters

### PR DESCRIPTION
**Pull request by elpamart**
clean and add pre-sets in osm download functions. Fix cut geom function and add extra filters.

### CODE REVIEW

File: osmUtils/utils_osm.py [hunk 1]
```diff
@@ -268,46 +271,46 @@ def cut_geom(polygon, N):
...
-#     #    print('no objects to concatenate')
-#     #    gdf = None
-#     #
-#     return list_dfs
+def get_cut_dfs(multi_pol, filters):
+    """
+    Iterates over a cutted geometry and retrieves the cutted parts
+    Parameters
+    ----------
+    multi_pol:
```
Comment (01painadam) [inline osmUtils/utils_osm.py]: Should this be a shapely MultiPolygon or a geojson..?

File: osmUtils/utils_osm.py [hunk 2]
```diff
@@ -345,17 +348,18 @@ def download_OSM(
...
-            query_str = f'{overpass_settings};({filters}(poly:"{polygon_coord_str}");>;);out;'
-            print(query_str)
-            print(f'Requesting data within polygon from API in {len(polygon_coord_str)} request(s)')
-            response_j = overpass_request(
-                        query_str, 
-                        timeout=timeout, 
-                        overpass_endpoint=overpass_endpoint
-                    )
-            response_json.append(response_j)
+        for filter_ in filters:
```
Comment (01painadam) [inline osmUtils/utils_osm.py]: for a temp object use `_<name>`, in this case `_filter`

File: osmUtils/utils_osm.py (hunk 1 above)
Comment (01painadam) [reply osmUtils/utils_osm.py]: Its actually a list of Polygons from the `cut_geom` function. Just make this explicit :)

**Review by 01painadam — CHANGES_REQUESTED** [review]

File: osmUtils/utils_osm.py (hunk 1 above)
Comment (01painadam) [reply osmUtils/utils_osm.py]: I've updated the docstring a little and renamed `multi_pol` to `polygon_list`

File: osmUtils/utils_osm.py (hunk 2 above)
Comment (elpamart) [reply osmUtils/utils_osm.py]: ok, it's changes now! :)

**Pull request merged**